docs/semgrep-code/supported-languages-python.md

```diff
@@ -0,0 +1,195 @@
+|  78 | stdlib3                  | Regex Library                            |
+|  79 | aws-lambda               | Serverless Framework                     |
+|  80 | aiohttp                  | Web Framework                            |
+|  81 | cherrypy                 | Web Framework                            |
+|  82 | django                   | Web Framework                            |
+|  83 | django-crispy-forms      | Web Framework                            |
+|  84 | django_allauth           | Web Framework                            |
+|  85 | django_channels          | Web Framework                            |
+|  86 | django_rest_frameworkapi | Web Framework                            |
+|  87 | fastapi                  | Web Framework                            |
+|  88 | flask                    | Web Framework                            |
+|  89 | flask-jwt-extended       | Web Framework                            |
+|  90 | flask-login              | Web Framework                            |
+|  91 | flask-session            | Web Framework                            |
+|  92 | flask-talisman           | Web Framework                            |
+|  93 | flask-wtf                | Web Framework                            |
+|  94 | lang                     | Web Framework                            |
+|  95 | pyramid                  | Web Framework                            |
+|  96 | starlette                | Web Framework                            |
+|  97 | wtforms                  | Web Framework                            |
+|  98 | libxml2                  | XML Parsing Library                      |
+|  99 | lxml                     | XML Parsing Library                      |
+| 100 | sax                      | XML Parsing Library                      |
+| 101 | stdlib                   | XML Parsing Library                      |
+| 102 | stdlib2                  | XML Parsing Library                      |
+| 103 | stdlib3                  | XML Parsing Library                      |
+| 104 | xml                      | XML Parsing Library                      |
+| 105 | xml.dom                  | XML Parsing Library                      |
+| 106 | xml.dom.minidom          | XML Parsing Library                      |
+| 107 | xml.dom.pulldom          | XML Parsing Library                      |
+| 108 | xml.etree                | XML Parsing Library                      |
+| 109 | xml.sax                  | XML Parsing Library                      |
+
+
+</details>
+
+### Benchmark results (_exclusive of_ [AI](https://semgrep.dev/docs/semgrep-assistant/overview) processing): 
+
+Our benchmarking process involves scanning open-source repositories, triaging the findings, and making iterative rule updates. This process was developed and is used internally by our security research team to monitor and improve rule performance.
+
+Results as of **2024-09-04**:
```

Comment:
```suggestion
Results as of **September 9, 2024**:
```
It is recommended by the google style guide to spell out [the date in full](https://developers.google.com/style/dates-times#why-dates-written-out) if we can:
![image](https://github.com/user-attachments/assets/3be7145d-716f-4908-89ef-be439b7fd13e)